### DISCUSSION TITLE
There was an error committing your changes: File could not be edited

### DISCUSSION THREAD

**Discussion by ghost** [community/community#66735] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

* https://github.com/jindam-vani/privacy-friendly-boardgame-clock/blob/master/README.md
* make few changes above file and try to commit, error_ **There was an error committing your changes: File could not be edited**

**Comment by seyLu** [comment, marked as answer]
Hi @jindam-vani,

In a collaborative setting, usually the default branch (`main`, or in this case `master`) is protected. What this means is that, the owner of the repo prolly disallow pushing directly to the main branch. What you can do though, is to create a new branch from `master`, work on that branch, add > commit > push as usual, then create and submit a Pull Request. Once the owner of the repo reviews your changes, and approves of it, and merge the changes to `master`, then and only then the changes will reflect to the original repo.

You can read more at the Github Official docs:
- [Github Flow](https://docs.github.com/en/get-started/quickstart/github-flow)
- [About Forks](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks)
- [About Permission and Visibility of Forks](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-permissions-and-visibility-of-forks)
- [About Branch Protection Rules](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches)
